tool/src/webapp/bbb-autoclose.html

```diff
@@ -26,23 +26,23 @@
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <script type="text/javascript" language="JavaScript" src="/bbb-tool/lib/jquery-1.3.2.min.js"></script> 
 <script type="text/javascript" language="JavaScript">
-$(document).ready(function(){
-    closeWindow(); closeWindowAlternative();
-});
+
 
 function closeWindow() {
-    var objWindow = window.open(location.href, "_self");
-    objWindow.close(); 
+    var myWindow = window.open('', '_self', '');
+    myWindow.close();
 }
 function closeWindowAlternative() {
     window.opener = 'bbb-tool';
     window.close();
 }
 </script>
 </head>
-<body style="width:99.5%">
-    <span style="margin:50px auto">
-        <a href="javascript:;" onclick="closeWindow();closeWindowAlternative();">Close this window</a>
+<body onload="closeWindow()" style="width:99.5%">
+    <span style="margin:50px auto; font-size: x-large">
+        <p>You left the meeting</p>
```

Comment:
The strings need to be localized

Reply:
I'm not quite sure how to implement that. Where do you tell bbb to redirect to that static html?